app/templates/administrative-units/index.hbs

```diff
@@ -10,14 +10,22 @@
           </p>
           <p>
             <AuLabel for="filter-municipality">Gemeente</AuLabel>
-            <AuInput
-              id="filter-municipality"
-              @width="block"
-              @value={{this.municipality}}
+            <MunicipalitySelect
+              @location="municipality"
+              @selected={{this.municipality}}
+              @set={{this.setMunicipality}}
```

Comment:
Nitpicky, but I prefer if actions have a more descriptive name. I usually go for `onChange` if I want to keep it generic.